runtime/compiler/runtime/MethodMetaData.c

```diff
@@ -30,10 +30,62 @@
 #include "rommeth.h"
 #include "env/jittypes.h"
 
+#if defined(DEBUG)
+#define JITINLINE
+#else
+#if defined(_MSC_VER)
+#define JITINLINE __forceinline
+#elif (((__GNUC__ > 3) || (__GNUC__ == 3 && __GNUC_MINOR__ >= 1)) || defined(__clang__)) && !defined(J9VM_GC_REALTIME)
+#define JITINLINE inline __attribute((always_inline))
+#else
+#define JITINLINE __inline
+#endif
+#endif
```

Comment:
Does `JITINLINE` == `VMINLINE`? If so I don't see the value of having this duplicate value. We should just use `VMINLINE` to be consistent.

Reply:
> Does JITINLINE == VMINLINE?

They are close but not identical.

> We should just use VMINLINE to be consistent.

Yep, I was going for a more minimal PR, but am more than happy to tidy up. Before we switch to `VMINLINE`, there is one strange detail here that I would like to understand. In this condition:

```c++
#elif (((__GNUC__ > 3) || (__GNUC__ == 3 && __GNUC_MINOR__ >= 1)) || defined(__clang__)) && !defined(J9VM_GC_REALTIME)
```

Why would it say `&& !defined(J9VM_GC_REALTIME)`? It seems important, but I don't see why it's here. `J9VM_GC_REALTIME` is always defined--so effectively, `JITINLINE` will always be `__inline`. Meanwhile, `VMINLINE` does something which makes more sense.

I'm going to try switching to `VMINLINE` and see if it everything works (I expect it will).

Reply:
Probably some historical artifact. Yeah I'd say if `VMINLINE` works after your tests we just switch to that and keep things simple. Thanks!